src/cmd-buildupload

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python3
+
+# The inverse of cmd-buildprep.
```

Comment:
This statement is confusing to me. Can you elaborate here?

Reply:
Expanded on this a bit!